templates/almalinux-kitten-10.yaml

```diff
@@ -0,0 +1,20 @@
+# This template requires Lima v0.11.1 or later.
+
+images:
+- location: "https://kitten.repo.almalinux.org/10-kitten/cloud/x86_64/images/AlmaLinux-Kitten-GenericCloud-10-20241227.0.x86_64.qcow2"
```

Comment:
Can we have an updater script?

Reply:
It seemed crazy complicated though

Reply:
cc @norio-nomura can you help this? 🙏

Reply:
* https://github.com/lima-vm/lima/issues/3447